### PR TITLE
refactor: move webpush to its own router

### PR DESCRIPTION
**Pull request by dni**
refactor to make `views/api.py` cleaner

### CODE REVIEW

**Review by prusnak — APPROVED** [review]
utACK

File: lnbits/core/views/webpush_api.py [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+    require_admin_key,
+)
+
+from ..crud import (
+    create_webpush_subscription,
+    delete_webpush_subscription,
+    get_webpush_subscription,
+)
+
+webpush_router = APIRouter(prefix="/api/v1/webpush", tags=["webpush"])
```
Comment (motorina0) [inline lnbits/core/views/webpush_api.py:27]: **Nitpick**: a bit inconsistent reported to `tinyurl_router` (which does not have a prefix):
```py
tinyurl_router = APIRouter()
```

Both `tinyurl_router` and `webpush_router` should [either have or not have a `prefix` ](https://github.com/lnbits/lnbits/pull/2154/files#diff-f2fbe70e0e85aafee5ab16b5ffa59e11acbafd33d2e7a606f94ca745018dd722R22)

I like with `prefix` better: normally the version is updated for all the paths in the file.

Comment (motorina0) [reply lnbits/core/views/webpush_api.py:27]: oh, I see 👍

**Review by motorina0 — APPROVED** [review]
utACK

**Pull request merged**